app/views/submission.scala.html

```diff
@@ -423,14 +433,15 @@ <h2 data-bind="text: abstract().title"></h2>
         <h3>Enter the Abstract Text</h3>
     </script>
     <script id="body-abstract-text-editor" type="text/html">
-        <textarea rows="10" class="form-control" id="text" maxlength="2000"
-        data-bind="value: editedAbstract().text, valueUpdate: 'afterkeydown'">
+        <textarea rows="10" class="form-control" id="text"
+                data-bind="value: editedAbstract().text, valueUpdate: 'afterkeydown'"
+                maxlength="@Model.getLimit[Abstract]("text")">
         </textarea>
         <div>
             <span>LaTeX-like equation typesetting is supported via MathJax. Use $$ and $ (inline).
-                <a target="_blank" href="http://meta.math.stackexchange.com/questions/5020/mathjax-basic-tutorial-and-quick-reference">Quick quide</a>
+                <a target="_blank" href="http://meta.math.stackexchange.com/questions/5020/mathjax-basic-tutorial-and-quick-reference">Quick guide</a>
             </span><br>
-            <span>Characters left: </span><span data-bind="text: editorTextCharactersLeft"></span>
+            <span data-bind="text: editorTextShowCharsLeft"></span><span data-bind="text: editorTextCharactersLeft"></span>
```

Comment:
As an alternative one could also use the `if` binding. This can forther be used to show an alternative text:

```html
<span data-bind="if: editorTextCharactersLeft">
    Chars left: <span data-bind="text: editorTextCharactersLeft"></span>
</span>
<span data-bind="ifnot: editorTextCharactersLeft">
    Please enter your abstract text</span>
</span>
```